.github/actions/spelling/patterns.txt

```diff
@@ -1,60 +1,85 @@
+(?:^|[\s=;:?])[-a-zA-Z=;:/0-9+]{50,}(?:[\s=;:?]|$)
+
+# hit-count: 10 file-count: 8
+# GitHub SHAs (markdown)
+(?:\[`?[0-9a-f]+`?\]\(https:/|)/(?:www\.|)github\.com(?:/[^/\s"]+){2,}(?:/[^/\s")]+)(?:[0-9a-f]+(?:[-0-9a-zA-Z/#.]*|)\b|)
+
+# hit-count: 10 file-count: 4
+# githubusercontent
+/[-a-z0-9]+\.githubusercontent\.com/[-a-zA-Z0-9?&=_\/.]*
+
+# hit-count: 9 file-count: 6
 # hex runs
 \b[0-9a-fA-F]{16,}\b
 
+# hit-count: 8 file-count: 5
+# base64 encoded content
+([`'"])[-a-zA-Z=;:/0-9+]+=\g{-1}
+
+# hit-count: 6 file-count: 3
+# tar arguments
+\b(?:\\n|)g?tar(?:\.exe|)(?:(?:\s+--[-a-zA-Z]+|\s+-[a-zA-Z]+|\s[ABGJMOPRSUWZacdfh-pr-xz]+\b)(?:=[^ ]*|))+
+
+# hit-count: 5 file-count: 3
+# alternate markers if you run into latex and friends
+(?<!\\)\\[nrt](?=[a-z]{2,})(?=.*['"`])
+
 # hit-count: 4 file-count: 2
 # mailto urls
 mailto:[-a-zA-Z=;:/?%&0-9+@.]{3,}
 
-# hit-count: 2 file-count: 1
-# IPv6
-\b(?:[0-9a-fA-F]{0,4}:){3,7}[0-9a-fA-F]{0,4}\b
+# hit-count: 4 file-count: 2
+# lower URL escaped characters
+\%[0-9a-f][a-f](?=[a-z]{2,})
 
-# hit-count: 1 file-count: 1
+# hit-count: 4 file-count: 1
 # Amazon
 \bamazon\.com/[-\w]+/(?:dp/[0-9A-Z]+|)
 
-# hit-count: 1 file-count: 1
-# Non-English
-[a-zA-Z]*[ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýÿĀāŁłŃńŅņŒœŚśŠšŜŝŸŽžź][a-zA-Z]{3}[a-zA-ZÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýÿĀāŁłŃńŅņŒœŚśŠšŜŝŸŽžź]*
+# hit-count: 2 file-count: 1
+# URL escaped characters
+\%[0-9A-F][A-F](?=[A-Za-z])
+
+# hit-count: 2 file-count: 1
+# IPv6
+\b(?:[0-9a-fA-F]{0,4}:){3,7}[0-9a-fA-F]{0,4}\b
+
+# hit-count: 2 file-count: 1
+# uuid:
+\b[0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}\b
 
-# Automatically suggested patterns
 # hit-count: 1 file-count: 1
 # oracle
 \bdocs\.oracle\.com/[-0-9a-zA-Z./_?#&=]*
 
-# GitHub SHAs
-\bapi.github\.com/repos/[^/]+/[^/]+/[^/]+/[0-9a-f]+\b
+# hit-count: 1 file-count: 1
+# Non-English
+[a-zA-Z]*[ÀÁÂÃÄÅÆČÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝßàáâãäåæčçèéêëìíîïðñòóôõöøùúûüýÿĀāŁłŃńŅņŒœŚśŠšŜŝŸŽžź][a-zA-Z]{3}[a-zA-ZÀÁÂÃÄÅÆČÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝßàáâãäåæčçèéêëìíîïðñòóôõöøùúûüýÿĀāŁłŃńŅņŒœŚśŠšŜŝŸŽžź]*|[a-zA-Z]{3,}[ÀÁÂÃÄÅÆČÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝßàáâãäåæčçèéêëìíîïðñòóôõöøùúûüýÿĀāŁłŃńŅņŒœŚśŠšŜŝŸŽžź]|[ÀÁÂÃÄÅÆČÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝßàáâãäåæčçèéêëìíîïðñòóôõöøùúûüýÿĀāŁłŃńŅņŒœŚśŠšŜŝŸŽžź][a-zA-Z]{3,}
+
 (?:\[[0-9a-f]+\]\(https:/|)/(?:www\.|)github\.com/[^/]+/[^/]+(?:/[^/]+/[0-9a-f]+(?:[-0-9a-zA-Z/#.]*|)\b|)
-\bgithub\.com/[^/]+/[^/]+[@#][0-9a-f]+\b
-# githubusercontent
-/[-a-z0-9]+\.githubusercontent\.com/[-a-zA-Z0-9?&=_\/.]*
-# gist github
-/gist\.github\.com/[^/]+/[0-9a-f]+
-# git.io
-\bgit\.io/[0-9a-zA-Z]+
-# GitHub JSON
-"node_id": "[-a-zA-Z=;:/0-9+]*"
-# Contributor
-\[[^\]]+]\(https://github\.com/[^/]+\)
-# GHSA
-GHSA(?:-[0-9a-z]{4}){3}
 
 x-amz-request-id:[0-9A-F]+
 x-amz-id-2:[-a-zA-Z=;:/0-9+]*
 
 # mailbox
-/mailboxes/[0-9a-f]+(?:/[0-9a-f]+|/idx/bundle\.[0-9a-f]+\.[0-9a-f]+|)
+/mailboxes/[0-9a-f]{2,}(?:/[0-9a-f]+|/idx/bundle\.[0-9a-f]+\.[0-9a-f]+|)
```

Comment:
this is kinda a bug-fix to the repository's pattern which was slicing words -- this new version of check-spelling would give a helpful warning about such things.